src/components/attribution-editor.js

```diff
@@ -35,27 +35,34 @@ class AttributionEditor extends React.Component {
 
     return el(
       'div'
-      , { className: 'AttributionEditor' }
+      , { className: 'AttributionEditor'
+        , style:
+          { borderRadius: '3px'
+          }
+        }
       , this.renderCover()
       , el(
         'div'
         , { className: 'AttributionEditor-metadata'
           , style:
-            { maxWidth: 800
+            { width: '90%'
+            , left: '5%'
+            , bottom: 0
```

Comment:
@jones possible to go `top: -3em` or something here, so metadata can extend down (see 2nd screenshot)?